src/game/client/swarm/c_asw_concommands.cpp

```diff
@@ -1122,4 +1122,19 @@ CON_COMMAND( make_game_friends_only, "Changes access for the current game to fri
 	g_pMatchFramework->GetMatchSession()->UpdateSessionSettings( pSettings );
 
 	ClientPrint( NULL, 8, "#L4D360UI_Lobby_Access_Changed_Friends" );
-}
\ No newline at end of file
+}
+
+
+extern ConVar cl_asw_laser_sight_color;
+void ASW_ChangeLaser_f(const CCommand& args)
+{
+	if (args.ArgC() >= 4)
+	{
+		engine->ClientCmd(VarArgs("cl_changelaser %d %d %d", atoi(args[1]), atoi(args[2]), atoi(args[3])));
```

Comment:
this could be something that happens in a convar change callback if `engine->IsConnected()`

Reply:
Is there a good way to know if client is currently in lobby screen?

Currently, I have code in "nb_main_panel.cpp" that detects changes, but only sends out updates every 250ms so it doesn't flood server everytime a new color is picked. 

I could do it that way, but I need a way to know if lobby screen is active so that it can handle the laser color convar changes without flooding.

Reply:
`	if (m_LastNetworkSentColor != cl_asw_laser_sight_color.GetColor() && gpGlobals->curtime > m_lastColorNetworkUpdate + 0.25f)
	{
		SetChangingLaserColor(cl_asw_laser_sight_color.GetColor());
		m_LastNetworkSentColor = cl_asw_laser_sight_color.GetColor();
		m_lastColorNetworkUpdate = gpGlobals->curtime;
	}`

Reply:
You could only update the convar when the mouse is released to reduce the amount of updates.

Reply:
I added a method to the client player class that gets if briefing panel pointer is nonzero.